### ISSUE TITLE
Allow xml properties to be set via commandline

### ISSUE THREAD

**Issue by mackaiver** [mackaiver/streams-cta#38]
This works well for the streams-runtime. I can do something like 

`java -jar strasm.jar test.xml -Dnum_copies=8`

If there is a corresponding xml property in the test.xml

`<property name="num_copies" value="1" />`

This does not work yet for streams-storm and streams-flink. 

This might simplify the makefile in PR #37

**Comment by alexeyegorov** [comment]
should the ```-Dnum_copies=8``` then overwrite the value of the ```property``` tag in the XML?

**Comment by mackaiver** [comment]
Yup. At least that's how it works with streams-runtime. It's quite useful.

**Comment by alexeyegorov** [comment]
ok. I will add this functionality to Flink, Storm and Spark.

**Comment by mackaiver** [comment]
![](http://s2.quickmeme.com/img/ce/ce68c60328bf7a4cf98700b75e40160cd64f8aba1d070a541a8b295849914522.jpg)

**Comment by alexeyegorov** [comment]
fixed: this should now be available after rebuilding the projects.

**Comment by mackaiver** [comment]
Does not seem to work btw. I rebuild the project several times. Do I need to update some dependencies?
